proto/metadata.proto

```diff
@@ -621,6 +621,9 @@ message Metadata {
     bool eth_channel_ingress_parameters_unsupported = 222;
     // Cisco numbering for eth channel assignment starts from 1 instead of 0
     bool eth_channel_assignment_cisco_numbering = 223;
+    // operational_mode_unsupported is set to true for devices that do not support optical_channel  leaf operational-mode
```

Comment:
Perhaps for conciseness:

// True when the optical-channel operational-mode is unsupported.